src/modules/powerrename/dll/PowerRenameExt.vcxproj.filters

```diff
@@ -53,18 +53,19 @@
       <Filter>Source Files</Filter>
     </None>
     <None Include="packages.config" />
+    <None Include="resource.base.h">
+      <Filter>Header Files</Filter>
+    </None>
+    <None Include="PowerRenameExt.base.rc">
+      <Filter>Resource Files</Filter>
+    </None>
+    <None Include="Resources.resx">
+      <Filter>Resource Files</Filter>
+    </None>
```

Comment:
These changes have been made by Visual Studio, you didn't do anything manual here, right?

Reply:
No, there is nothing manual. I didn't understand why this changes are there though.

Reply:
OK, no problem.